frontend/src/ofrak/remote_resource.js

```diff
@@ -82,7 +83,7 @@ export class RemoteResource extends Resource {
     super(resource_id, data_id, parent_id, tags, caption, attributes);
 
     this.resource_list = resource_list;
-    this.uri = `/${this.resource_id}`;
+    this.uri = this.resource_id;
```

Comment:
@EdwardLarson See [this discussion](https://github.com/redballoonsecurity/ofrak/pull/255#discussion_r1137816449).

Reply:
That discussion doesn't really explain why the change was made the way it was made or why one would even want to do it that way. We made a variable so that if we make general changes to the backend request URL, we change only that one place. Then for some reason made the change in every place the variable is used and turned the "uri" variable into just the resource ID without even changing the name.

Reply:
I'm just pointing out that I suggested it, and that Dan presumably had some reason for choosing to do it his way :)

In general, I agree with your proposed change.